### DISCUSSION TITLE
Is there some kind of config generator/constructor to visual create layout?

### DISCUSSION THREAD

**Discussion by Makishima** [microsoft/terminal#12486] [Q&A]
Hello!
I'm wonder if I can create startup layout in the some kind of constructor? 
I need to create few tabs each of it will have split panes, different startup path and profiles.
Using `startupActions` in the json config a bit pain for now.

**Comment by Makishima** [comment]
Any ideas? :D

**Reply by zadjii-msft** [reply]
I'm not aware of any such tools at the moment. `persistWindowLayout` in the Preview builds will automatically serialize the Terminal state to a json blob of actions, but not as a `wt` commandline that can be used for `startupActions`. Maybe that code wouldn't be too hard to write?
